core/src/i18n/en.yaml

```diff
@@ -46,12 +46,34 @@ cluster-role-bindings:
 cluster-roles:
   title: ClusterRoles
 clusters:
+  add:
+    title: Add Cluster
+  common:
+    api-server-address: API server address
+    download-kubeconfig: Download kubeconfig
+    kubeconfig-download-error: Failed to download the Kubeconfig
+    kubeconfig-not-present: Kubeconfig is missing on the Cluster
   list:
     outdated:
       download: Download Kubeconfig
       tooltip: Outdated parameter version may cause errors. Delete and re-add your cluster.
+    no-clusters-found: No clusters found
   overview:
     title: Cluster Overview
+    version: Version
+  storage:
+    changed-message: Cluster storage changed from {{originalStorage}} to {{targetStorage}}.
```

Comment:
I think we add more info on why it happened. Something similar to `The chosen storage type has been overwritten by the Cluster configuration from {{originalStorage}} to {{targetStorage}}.`